### DISCUSSION TITLE
Hey does this work for Anyone Else

### DISCUSSION THREAD

**Discussion by v1peridae** [giorgimode/SpotMyStatus#57] [General]
Doesn't seem to work for me but I'm not sure if it's just a me issue or it's for everyone else. The website also seems to be down? Love this app sm :')

**Comment by giorgimode** [comment]
hey @v1peridae server currently struggles with the load during the peak times. I need to migrate it to a more stable environment.

**Reply by v1peridae** [reply]
Ohh I see! Thanks!
